src/Language/DifferentialDatalog/Parse.hs

```diff
@@ -78,7 +78,6 @@ reservedNames = ["as",
                  "bool",
                  "break",
                  "continue",
-                 "default",
```

Comment:
didn't realize this was a keyword. was it used anywhere?

Reply:
No, it was there by mistake.